<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CKANBrowserDialogDataProviders</class>
 <widget class="QDialog" name="CKANBrowserDialogDataProviders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>dlg_dataproviders_title</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QgsCollapsibleGroupBox" name="IDC_grpManualDataProvider">
     <property name="title">
      <string>dlg_dataproviders_grp_manual</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="IDC_lbl_enter_dataprovider_url">
        <property name="text">
         <string>dlg_dataproviders_lbl_enter_dataprovider_url</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="IDC_leManualUrl"/>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="IDC_bTestConnection">
           <property name="text">
            <string>dlg_dataproviders_btn_test_connection</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IDC_bAddConnection">
           <property name="text">
            <string>dlg_dataproviders_btn_add_connection</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>dlg_dataproviders_lbl_search_for_provider</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="IDC_lbInstanceCount">
     <property name="text">
      <string>dlg_dataproviders_lbl_nr_of_instances</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="IDC_listProviders">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CKANBrowserDialogDataProviders</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>735</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CKANBrowserDialogDataProviders</receiver>
   <slot>searchTermChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CKANBrowserDialogDataProviders</receiver>
   <slot>save_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>733</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>750</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_bTestConnection</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogDataProviders</receiver>
   <slot>btn_clicked_test_connection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>IDC_bAddConnection</sender>
   <signal>clicked()</signal>
   <receiver>CKANBrowserDialogDataProviders</receiver>
   <slot>btn_clicked_add_connection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>searchTermChanged(QString)</slot>
  <slot>save_btn_clicked()</slot>
  <slot>btn_clicked_test_connection()</slot>
  <slot>btn_clicked_add_connection()</slot>
 </slots>
</ui>
